gulpfile.js

```diff
@@ -129,7 +131,21 @@ defineBuildTask('HTMLImports');
 defineBuildTask('ShadowDOM');
 defineBuildTask('MutationObserver');
 
-gulp.task('build', ['webcomponents', 'webcomponents-lite', 'CustomElements', 
+gulp.task('CustomElementsV1', function () {
+  return gulp.src('./src/CustomElements/v1/CustomElements.js', {base: './'})
+      .pipe(closureCompiler({
```

Comment:
@addyosmani Since I hadn't seen or worked with the rest of the polyfills, I decided during development to just focus on a from scratch and simpler (to me anyway) build workflow that gets us maximum optimizations. As we get closer to releasing this, I'm fully in favor of integrating better.

I still haven't fully ingested the custom build system used in the rest of the polyfills. I'd personally be more inclined to move everything to Closure and not have to maintain something custom. I'm not sure how much work that is at the moment.